src/flb_input.c

```diff
@@ -141,6 +140,23 @@ int flb_input_log_check(struct flb_input_instance *ins, int l)
     return FLB_TRUE;
 }
 
+/* Prepare input co-routines for the thread. */
+void flb_input_prepare()
```

Comment:
we always use init/exit , start/stop, create/destroy, please use init/exit to avoid adding a new option

Reply:
flb_output also uses `flb_output_prepare`.

Reply:
@edsiper should I break the convention, leave it as is or also update flb_output?

Reply:
I decided to follow the init/exit convention over the convention for output, @edsiper.